helm/cas-ciip-portal/templates/mailhog-route.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.mailhog.enable }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{ template "cas-ciip-portal.fullname" . }}-mailhog
+  labels:
+{{ include "cas-ciip-portal.labels" . | indent 4 }}
+
+spec:
+  host: cas-mailhog-{{ .Values.route.host }}
```

Comment:
actually, we may want to add a `mailhog.host` in our `values`, since `route.host` is a gov.bc.ca address (instead of pathfinder.gov.bc.ca)